index.js

```diff
@@ -1,22 +1,60 @@
 #!/usr/bin/env node
-// npm install --save-dev uglify-js@github:mishoo/UglifyJS2#harmony
-const bundler = require('./bundler');
+const { ArgumentParser } = require('argparse');
+const { version } = require('./package.json');
+const Bundler = require('./bundler');
 
-const args = process.argv.slice(2)
-/** Args
- * args[0] -- html file path
- * TODO: args[1] -- assets folder path
- * TODO: args[2] -- assets folder restination path
- * TODO: args[3] -- custom CSS tags
- * TODO: args[4] -- custom JS tags
- */
+const PATH_CONFIG = {
+  HTML: 'index.html',
+  DIST: 'dist',
+  ASSETS: 'assets',
+  CSS: 'css',
+  JS: 'js',
+  FOLDER: '',
```

Comment:
Can we rename "FOLDER" to "SOURCE_FOLDER" here and in the args `help` message? Because it's not so obvious what folder it is about.